### ISSUE TITLE
Return `lastEpisodeTime` on `/search/byterm` (and other search) endpoint(s)

### ISSUE THREAD

**Issue by keunes** [Podcastindex-org/api#3]
Hello,

In AntennaPod we got this [request](https://forum.antennapod.org/t/search-setting-suggestion/2549):
> When I search for a podcast I often find podcasts that are old and inactive. It would be awesome to search for podcasts that have at least released a podcast in the last 30 days.
>> It would be great if the date of the most recent episode was shown in the search results, and that the search results could be sorted by most recent episode date.

The [`/search/byterm`](https://podcastindex-org.github.io/docs-api/#get-/search/byterm) endpoint already provides `lastUpdateTime`. To fulfil the user's request, it would be helpful if it would also return a `lastEpisodeTime`. Ideally this would be the the most recent `pubDate` from all the episodes (items) in the feed, but if not, maybe the value of the first episode (top item) which in most cases would be the same item.

**Comment by daveajones** [comment]
Hmm. You just want a way to limit the search results returned by last publish time. I can probably figure that out.

**Comment by keunes** [comment]
No, sorry, I don't think our end-user requested to limit the search results. They just want to have, per podcast, one piece of extra information displayed in the UI: the release date of the most recent episode.

**Comment by daveajones** [comment]
Oh I see. Yes that’s very doable.

**Comment by daveajones** [comment]
Done!
![image](https://user-images.githubusercontent.com/1498236/207971227-5708586e-a642-4509-b124-b1fbb0c506e9.png)

**Comment by keunes** [comment]
Oh my. So fast. Thanks a lot!

**Comment by daveajones** [comment]
Going to reopen this because Steven let me know that this naming doesn't match a similar property in another endpoint response.  Going to fix that. Hang tight...
